docs/relations/partner.mdx

```diff
@@ -1,16 +1,8 @@
 ---
 sidebar_position: 1
-sidebar_label: Approved Partner Program
+sidebar_label: Betaflight Partners
 ---
 
-import PartnerText from './../../docs/development/manufacturer/approved-partner-program.md'
-import ThemedImage from '@theme/ThemedImage'
-import useBaseUrl from '@docusaurus/useBaseUrl'
-
-# Approved Partner Program
```

Comment:
@SupaflyFPV , i think you need to keep a header. e.g. `# Betaflight Partners` to fix the title text you questioned.